src/content/changelog/4-97-0-2021-06-25.md

```diff
@@ -0,0 +1,25 @@
+---
+title: '4.97.0: 2021-06-25'
+date: 2021-06-25T04:00:00.000Z
+version: 4.97.0
+changelog:
+  - API
+---
+
+### Added
+
+- Manage your Account's payment options with the new Payment Method Add ([POST /account/payment-methods](https://www.linode.com/docs/api/account/#payment-method-add)) endpoint.
+
+- Promo Codes for new signups can now be redeemed with the Promo Credit Add ([POST /account/promo-codes](https://www.linode.com/docs/api/account/#promo-credit-add)) endpoint. Previously, Promo Codes were entered only while signing up for a new account.
+
+- You can now delete individual LKE Nodes with the new Node Delete ([DELETE /lke/clusters/{clusterId}/nodes/{nodeId}](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#node-delete)) endpoint.
+
+### Changed
+
+- The Credit Card Add/Edit ([POST /account/credit-card](https://www.linode.com/docs/api/account/#credit-card-addedit)) has been deprecated. Please use the new Payment Method Add ([POST /account/payment-methods](https://www.linode.com/docs/api/account/#payment-method-add)) endpoint to manage your Account's credit card information.
```

Comment:
the url text link seems off - _#credit-card-addedit_

Reply:
looks like the `/` from `Add/Edit` isn't rendered as a hyphen as expected, but that's how the site seems to currently handle it 🤷 (the link directs to the correct section)

I can submit a bug report to change that

Reply:
ahh I see. Add Edit. ok